writer/licence_writer.go

```diff
@@ -34,6 +34,10 @@ func CheckIfLicenceExists(fileName string) (bool, error) {
 	for scanner.Scan() {
 		fileContent += scanner.Text() + "\n"
 	}
+	err2 := file.Close()
```

Comment:
Why creating new error variable instead of using the one that you have already created. 
Line 28, `file, err := ....`